src/box/memtx_tx.c

```diff
@@ -3001,4 +3012,5 @@ memtx_tx_snapshot_cleaner_destroy(struct memtx_tx_snapshot_cleaner *cleaner)
 {
 	if (cleaner->ht != NULL)
 		mh_snapshot_cleaner_delete(cleaner->ht);
+	memtx_tx_story_gc();
```

Comment:
Since we does not create/delete stories here perhaps we should not collect them too.

Reply:
I added it just for the sake of more often garbage collection.

Okay, removed it here for clarity. Perhaps we could move the snapshot cleaner stuff into a separate file?